.github/workflows/deploy.yml

```diff
@@ -6,9 +6,12 @@ on:
     types: [published]
 
 jobs:
+  test:
+    uses: ./.github/workflows/test.yml
   build:
     name: Create extension
     runs-on: ubuntu-18.04
```

Comment:
Github will deprecate the ubuntu-18.04 runner image pretty soon.
 https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/
I thkink you could do the updates in one fell swoop.

Same for the test.yml

Reply:
Updated, and updated the various Actions we're using as well.

Reply:
Thanks, looks good to me.